### PR TITLE
Added mean_wind_speed @the-louie suggest

### PR DESCRIPTION
**Pull request by helto4real**
I added a new PR that replaces #1 . It was easier to do that cause of changes that makes merges difficult. Only difference is that I kept the old wind_speed for backwards compability and added the mean_wind_speed as @the-louie suggested in that PR.

@the-louie would you mind check the code that it seems like you intended please?

### CODE REVIEW

**Comment by the-louie** [comment]
Thanks! Now i just have to remember why I needed it :)

File: smhi/smhi_lib.py [hunk 1]
```diff
@@ -398,6 +410,7 @@ def _get_all_forecast_from_api(api_result: dict) -> OrderedDict:
             precipitation,
             wind_direction,
             wind_speed,
+            wind_speed,
```
Comment (the-louie) [inline smhi/smhi_lib.py:413]: Is wind_speed included twice?

Comment (helto4real) [reply smhi/smhi_lib.py:413]: It is kinda flunky design hehe but there are only one wind speed from the original api call so I enter that into the object but one of them will be turned into mean_wind_speed. I probably could design that better..

Comment (helto4real) [reply smhi/smhi_lib.py:413]: Will not merge this until there are demand for it

**Comment by helto4real** [comment]
closing this since interests of adding feature is not present

**Pull request closed without merging**